templates/kubernetes/terraform/modules/kubernetes/backend_service.tf

```diff
@@ -0,0 +1,24 @@
+data "aws_secretsmanager_secret" "by-name" {
+  name = "<% .Name %>_cf_keypair"
+}
+
+data "aws_secretsmanager_secret_version" "cf_keypair" {
+  secret_id = data.aws_secretsmanager_secret.<% .Name %>_cf_keypair.id
+}
+
+locals {
+  cf_keypair_json = jsondecode(data.aws_secretsmanager_secret_version.cf_keypair.secret_string)
+}
+
+resource "kubernetes_secret" "cf_keypair" {
+  metadata {
+    name = "cf-keypair"
```

Comment:
This needs to specify the namespace to create the secret in. In this case it should be `namespace = kubernetes_namespace.app_namespace.metadata[0].name`